### DISCUSSION TITLE
Wrong file version served

### DISCUSSION THREAD

**Discussion by arsu-leo** [cdnjs/cdnjs#14041] [General] [answered]
Hi,

I'm unable to find where to report this.
Cdnjs is providing a wrong file version:
https://cdnjs.com/libraries/snap.svg
both 0.5.1 and 0.5.0 returns the 0.5.0 files

Thank you

**Comment by MattIPv4** [comment, marked as answer]
It looks like this version was added by the bot automatically: https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/snap.svg/0.5.1

PeterBot added the files for the version on the 6th of February, but the files in the source repo with the 0.5.1 tag are from the 7th of February with a commit named "Regenerate dist", so I think it is likely the case that the tag was originally added for 0.5.1 whilst the 0.5.0 assets were still present, which we then picked up before it was fixed the next day: https://github.com/adobe-webplatform/Snap.svg/tree/v0.5.1
